README.md

```diff
@@ -111,6 +111,7 @@ Additionally, the following parameters can be used for additional control over t
     * Adding the `JUMP THE QUEUE` label to a PR will make that PR first in the queue immediately.
     * When a PR is merged, it automatically updates the first-queued PR with the default branch.
  * You can also pass `login` and `slack_webhook_url` to notify the PR author when they are in the 1st position of the merge queue.
+ * You can pass `auto_merge` as false to disable auto-merge for a PR when it is in the 1st position of the merge queue.
```

Comment:
might be more intuitive to do something like `no_auto_merge`, since i feel like the default behavior of any configurable boolean parameters should be false

Reply:
Is this a breaking change for those that already use it?